src/app/story/storyrow/storyrow.component.css

```diff
@@ -18,14 +18,7 @@
     position: relative;
     /* Vertically center the text */
     top: -8px;
-}
-
-.node-display-name {
-    font-weight: bold;
-}
-
-.node-timestamp {
-    font-size: 1.1rem;
+    width: 15.5rem;
```

Comment:
Could you please add a comment explaining why the width is necessary? I imagine we'll forget in a year from now.